Flow.Launcher/ViewModel/MainViewModel.cs

```diff
@@ -871,7 +871,7 @@ public void ResetPreview()
         {
             if (Settings.AlwaysPreview)
             {
-                ShowPreview();
+                ShowInternalPreview();
```

Comment:
When `AlwaysPreview` is true, specify internal preview.

Reply:
This is because QL is not suitable for using with always preview right?

Reply:
> This is because QL is not suitable for using with always preview right?

Yep

Reply:
What happens when we have other preview tools that is suitable?

Reply:
> What happens when we have other preview tools that is suitable?

Well maybe pass always preview to plugin and let plugin decide? If plugin returns false then use internal preview.

Reply:
Done. Let me know if good to merge.